### PR TITLE
Update AD GlobalalertTemplate + add feature toggle flag for intrusion detection

### PR DESCRIPTION
**Pull request by freecaykes**
## Description

Add more AD GlobalAlertTemplate matching the docs

## For PR author

- [ ] Tests for change.
- [ ] If changing pkg/apis/, run `make gen-files`
- [ ] If changing versions, run `make gen-versions`

## For PR reviewers

A note for code reviewers - all pull requests must have the following:

- [ ] Milestone set according to targeted release.
- [ ] Appropriate labels:
  - `kind/bug` if this is a bugfix.
  - `kind/enhancement` if this is a a new feature.
  - `enterprise` if this PR applies to Calico Enterprise only.

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -37,6 +37,7 @@ require (
 replace (
 	github.com/Azure/go-autorest => github.com/Azure/go-autorest v13.3.2+incompatible // Required by OLM
 	github.com/operator-framework/operator-sdk => github.com/operator-framework/operator-sdk v1.0.1
+	github.com/tigera/api => github.com/freecaykes/api v0.0.0-20220324232822-91c92879135e
```
Comment (freecaykes) [inline go.mod:40]: - [ ] FIXME: wait for api sync

File: pkg/render/intrusion_detection.go [hunk 2]
```diff
@@ -33,7 +33,6 @@ import (
 	operatorv1 "github.com/tigera/operator/api/v1"
 	"github.com/tigera/operator/pkg/components"
 	"github.com/tigera/operator/pkg/dns"
-	"github.com/tigera/operator/pkg/render/common/elasticsearch"
 	relasticsearch "github.com/tigera/operator/pkg/render/common/elasticsearch"
```
Comment (doublek) [inline pkg/render/intrusion_detection.go:36]: why change this?

Comment (freecaykes) [reply pkg/render/intrusion_detection.go:36]: oh extra cleanup switching over to the one defined below 
` relasticsearch "github.com/tigera/operator/pkg/render/common/elasticsearch" `

**Review by doublek — APPROVED** [review]
LGTM

**Review by Brian-McM — APPROVED** [review]

**Comment by marvin-tigera** [comment]
OK, I will merge the pull request when it's ready, leave the commits as is when I merge it, and delete the branch after I've merged it.

**Comment by marvin-tigera** [comment]
I'm sorry but I failed delete the branch after merging the pull request.

**Pull request merged**